appengine/flexible/endpoints/README.md

```diff
@@ -93,3 +93,70 @@ To use the client ID for authentication:
 Now you can use the client ID to make requests to the API:
 
     $ python clients/google-id-token-client.py https://YOUR-PROJECT-ID.appspot.com YOUR-API-KEY /path/to/client-id.json
+
+### Using the App Engine default service account client (no key file needed)
+
+The App Engine default service account client demonstrates how to use the Google App Engine default service account to authenticate to endpoints.
+We refer to the project that serves API requests as the server project. You also need to create a client project in the [Cloud Console](https://console.cloud.google.com).
+Both server and client projects are running Google App Engine standard applications.
```

Comment:
Is the server application running on standard? The server sample (in the directory above) uses Flexible.